### ISSUE TITLE
Overlay should be inert when modal is open on mobile screen readers

### ISSUE THREAD

**Issue by hrb-hub** [tutao/tutanota#8166]
Related to #7998

**Estimated: 0.25h**
**Spent: 0.25h**

## Test notes:
With either VoiceOver or TalkBack:
- [ ] Open mail editor, write something and close it. This should cause a "draft saved" overlay to appear.
- [ ] Without dismissing the overlay open the mail editor again (write a new email).
- [ ] Keep selecting next item until you reach the end.
- [ ] See that focus is trapped in the mail editor modal.
